evap/evaluation/models.py

```diff
@@ -1177,7 +1177,7 @@ def create_superuser(self, username, password, email=None, first_name=None, last
 class UserProfile(AbstractBaseUser, PermissionsMixin):
     username = models.CharField(max_length=255, unique=True, verbose_name=_('username'))
 
-    # null=True because users created through kerberos logins and certain external users don't have an address.
+    # null=True because certain external users don't have an address
```

Comment:
perhaps keep that comment because getting the email is not guaranteed?

Reply:
we discussed yesterday that the username is created from the email, so if we don't get the email from openid, we can't create the userprofile at all.
i think it should be safe to assume to get the email. if that should turn out to be wrong, we have to make some further changes and handle this case explicitly.